mono/utils/mono-error-internals.h

```diff
@@ -164,7 +164,7 @@ void
 mono_error_set_argument_format (MonoError *error, const char *argument, const char *msg_format, ...) MONO_ATTR_FORMAT_PRINTF(3,4);
 
 void
-mono_error_set_argument (MonoError *error, const char *argument, const char *msg, ...) MONO_ATTR_FORMAT_PRINTF(3,4);
+mono_error_set_argument (MonoError *error, const char *argument, const char *msg, ...);
```

Comment:
It shoudln't be variadic, either.  
```suggestion
mono_error_set_argument (MonoError *error, const char *argument, const char *msg);
```
Also need to update the definition to remove `...`
https://github.com/mono/mono/blob/bf7faa00ad7590ee7aebf3e06a43b1439bdaac64/mono/utils/mono-error.c#L546